test/MdnProvider.spec.js

```diff
@@ -0,0 +1,12 @@
+import DetermineTargetsFromConfig, { Versioning } from '../src/Versioning';
+import { getUnsupportedTargets } from '../src/providers/MdnProvider';
+
+describe('MdnProvider', () => {
+  it('should support Safari TP', () => {
+    const node = { protoChainId: 'AbortController' };
```

Comment:
Using `AbortController` which would have triggered #284 , but the actual capabilities of Safari TP can change over time, and the test doesn't necessarily work in the future. Any suggestions?

Reply:
AbortController is in regular Safari, so it's not ever going to disappear from TP.

Reply:
Great.